plugins/catalog-backend/src/actions/createUnregisterCatalogEntitiesAction.ts

```diff
@@ -25,51 +25,77 @@ export const createUnregisterCatalogEntitiesAction = ({
   actionsRegistry: ActionsRegistryService;
 }) => {
   actionsRegistry.register({
-    name: 'unregister-catalog-entities',
-    title: 'Unregister Catalog Entities',
+    name: 'unregister-entity',
+    title: 'Unregister entity from the Catalog',
     attributes: {
       destructive: true,
       readOnly: false,
       idempotent: true,
     },
-    description: `Unregister a Location and the entities it ownws.
+    description: `Unregisters a Location entity and all entities it owns from the Backstage catalog.
 
-This tool is analogous to the "unregister location" function you see in the Backstage dashboard,
-where you supply the UUID for a Location entity link that allows for the removal of the Location and
-the various Entities (Components, Systems, Resources, APIs, Users, and Groups) that were also created 
-when the Location was imported.
+This action is similar to the "Unregister location" function in the Backstage UI, where you provide the unique identifier (locationId) of a Location entity. The action will remove the specified Location from the catalog as well as all entities that were created when the Location was imported.
```

Comment:
I was able to validate both location id and location url with the new schema as well @benjdlambert ... should we not list both options in the tool description here?